### PR TITLE
Update the int8v2 C-API MKL-DNN doc

### PR DESCRIPTION
**Pull request by bingyanghuang**
Since in Release 1.5, MKL-DNN post-training quantization C-API has supported 4 more models compared with Release 1.4, here we update the readme for QA test.

### CODE REVIEW

**Comment by bingyanghuang** [comment]
@Sand3r- @wojtuss Please help review this updated readme.

File: paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md [hunk 1]
```diff
@@ -59,12 +70,22 @@ Notes:
...
 ```bash
 ./paddle/fluid/inference/tests/api/test_analyzer_int8_resnet50 --infer_model=third_party/inference_demo/int8v2/resnet50/model --infer_data=/path/to/converted/int8_full_val.bin --batch_size=1 --paddle_num_threads=1
 ```
-   * ##### Mobilenet-v1 Full dataset benchmark
+   * ##### Other 6 models Full dataset benchmark
 ```bash
-./paddle/fluid/inference/tests/api/test_analyzer_int8_mobilenet --infer_model=third_party/inference_demo/int8v2/mobilenet/model --infer_data=/path/to/converted/int8_full_val.bin --batch_size=1 --paddle_num_threads=1
+./paddle/fluid/inference/tests/api/test_analyzer_int8_resnet50 --infer_model= --infer_data=/path/to/converted/int8_full_val.bin --batch_size=1 --paddle_num_threads=1
 ```
+Set argument `--infer_model` with the following values in command line for other 6 models' benchmark:
```
Comment (Sand3r-) [inline paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md]: `for other 6 models' benchmark:` -> `to benchmark 6 other models:`

File: paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # INT8 MKL-DNN quantization 
 
-This document describes how to use Paddle inference Engine to convert the FP32 model to INT8 model on ResNet-50 and MobileNet-V1. We provide the instructions on enabling INT8 MKL-DNN quantization in Paddle inference and show the ResNet-50 and MobileNet-V1 results in accuracy and performance.
+This document describes how to use Paddle inference Engine to convert the FP32 model to INT8 model on following 7 models: ResNet-50, ResNet-101, MobileNet-V1, MobileNet-V2, VGG16, VGG19 and GoogleNet. We provide the instructions on enabling INT8 MKL-DNN quantization in Paddle inference and show all the 7 models' results in accuracy and performance.
```
Comment (Sand3r-) [inline paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md]: `and show all the 7 models' results in accuracy and performance` -> `and show the accuracy and performance results of all 7 models`.

File: paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md (hunk 1 above)
Comment (bingyanghuang) [reply paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md]: Done

File: paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md (hunk 2 above)
Comment (bingyanghuang) [reply paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md]: Done

**Pull request closed without merging**